packages/runtime/src/ax.tsx

```diff
@@ -19,23 +19,22 @@
  *
  * @param classes
  */
-export const ax = (classes: (string | undefined | false)[]): string => {
-  const found: Record<string, string> = {};
+export default function ax(classNames: (string | undefined | false)[]): string {
+  const atomicGroups: Record<string, string> = {};
+  let i = -1;
 
-  for (let i = 0; i < classes.length; i++) {
-    const cls = classes[i];
-    if (!cls) {
+  while (++i < classNames.length) {
+    if (!classNames[i]) {
       continue;
     }
 
-    const groups = cls.split(' ');
+    const groups = (classNames[i] as string).split(' ');
+    let x = -1;
 
-    for (let x = 0; x < groups.length; x++) {
-      const className = groups[x];
-      const group = className.slice(0, className.charCodeAt(0) === 95 ? 5 : undefined);
```

Comment:
We have removed `_` check. This might fail in following scenario:

```ts
// Will return 'foobar2 bar` but should be 'foobar1 foobar2 bar` 
ax(['foobar1 foobar2', 'bar']);
```

Reply:
hmm yeah true i'll add it back

Reply:
added back in - size back up to 99B but at least its under 100!